src/main/java/com/cloudbees/jenkins/support/api/LaunchLogsFileContent.java

```diff
@@ -23,14 +21,8 @@ public LaunchLogsFileContent(String name, String[] filterableParameters, File fi
 
     @Override
     protected InputStream getInputStream() throws IOException {
-        try (FileInputStream inputStream = new FileInputStream(file)) {
-            List<String> strings = IOUtils.readLines(inputStream, Charset.defaultCharset());
-            byte[] bytes = strings.stream()
-                    .map(line -> PasswordRedactor.get().redact(line))
-                    .collect(Collectors.joining("\n", "", "\n"))
-                    .getBytes(Charset.defaultCharset());
-            return new ByteArrayInputStream(bytes);
-        }
+        Function<String, String> filter = line -> PasswordRedactor.get().redact(line);
```

Comment:
May be slightly more efficient to avoid calling `lookupSingleton` for each line:
```suggestion
        Function<String, String> filter = PasswordRedactor.get()::redact;
```